src/components/pages/MatchUp/RenderMatchUp.js

```diff
@@ -11,139 +9,125 @@ import { modalInstructions } from '../../../utils/helpers';
+              <div id="fourth-player" className="player-image"></div>
+              <figcaption>player name</figcaption>
+            </div>
+          </div>
         </div>
-        <div className={'under-construction-content'}>
-          <h1>Under Construction!</h1>
-          <h1>Match Up UI Coming Soon!</h1>
-        </div>
-        <Button
-          style={{ margin: '1rem' }}
-          className="back-btn"
-          onClick={goToPointShare}
-        >
-          Next to Point Share
-        </Button>
       </div>
-      {/* REMOVE ONCE DESIGN WORK BEGINS - END */}
-      {/* <QuestionCircleOutlined
-        className="question-icon"
-        onClick={() => {
-          setModalVisible(true);
-        }}
-      />
-      {modalVisible && (
-        <InstructionsModal
-          modalVisible={modalVisible}
-          handleCancel={() => {
-            setModalVisible(false);
-          }}
-          handleOk={() => {
-            setModalVisible(false);
-          }}
-          instructions={modalInstructions.matchUp}
-        />
-      )} */}
-      {/* <div className="matchup-container">
-        <Row className="toprow">
-          <Col className="green-box" xs={24} sm={13}>
-            {faceoffs[0] && (
-              <FaceoffContent
-                custom_date={props.custom_date}
-                content={faceoffs[0]}
-                backgroundColor={'#C9E952'}
-                votesRemaining={props.votesRemaining}
-                votesNeededToUnlock={7}
-                dayNeededToUnlock={5}
-                hourNeededToUnlock={18}
-              />
-            )}
-          </Col>
-          <Col className="red-box" xs={24} sm={11}>
-            {faceoffs[1] && (
-              <FaceoffContent
-                custom_date={props.custom_date}
-                content={faceoffs[1]}
-                backgroundColor={'#e97451'}
-                votesRemaining={props.votesRemaining}
-                votesNeededToUnlock={7}
-              />
-            )}
-          </Col>
-        </Row>
-        <Row className="bottomrow">
-          <Col className="yellow-box" xs={24} sm={11}>
-            {faceoffs[2] && (
-              <FaceoffContent
-                custom_date={props.custom_date}
-                content={faceoffs[2]}
-                backgroundColor={'#ffde3b'}
-                votesRemaining={props.votesRemaining}
-                votesNeededToUnlock={8}
-              />
-            )}
-          </Col>
-          <Col className="blue-box" xs={24} sm={13}>
-            {faceoffs[3] && (
-              <FaceoffContent
-                custom_date={props.custom_date}
-                content={faceoffs[3]}
-                backgroundColor={'#438eac'}
-                votesRemaining={props.votesRemaining}
-                votesNeededToUnlock={9}
-              />
-            )}
-          </Col>
-        </Row>
-        <Button className="back-button" onClick={back2Dash}>
-          Back
-        </Button>
```

Comment:
Thank you for removing all of the old dead component from the file and making it do the thing.